gateway/helm/values.yaml

```diff
@@ -11,7 +11,7 @@ image:
   repository: ghcr.io/grafbase/gateway
 
   # Image tag to deploy. Default: latest.
-  tag: latest
+  tag: 0.27.1
```

Comment:
See my comments in https://github.com/grafbase/grafbase/pull/2527/files#r1926804701
For me it's unusual to have individual helm charts with minor releases for the application